core/src/test/java/org/infinispan/persistence/CacheAPIPersistenceTest.java

```diff
@@ -0,0 +1,54 @@
+package org.infinispan.persistence;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Collections;
+import java.util.Spliterators;
+
+import org.infinispan.api.APINonTxTest;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.container.DataContainer;
+import org.infinispan.container.DataContainer.ComputeAction;
+import org.infinispan.container.InternalEntryFactory;
+import org.infinispan.container.InternalEntryFactoryImpl;
+import org.infinispan.persistence.dummy.DummyInMemoryStoreConfigurationBuilder;
+
+/**
+ * Test that ensure that when persistence is used with an always empty data container that various operations
+ * are properly supported
+ * @author wburns
+ * @since 9.2
+ */
+public class CacheAPIPersistenceTest extends APINonTxTest {
```

Comment:
I'd rename it to `APINonTxPersistenceTest` to match the base class. Or maybe rename the base class?

Reply:
Okay.